[Bug 17851] Add a default script to scrollbars

dunbarx opened the pull request:
Broke long line into multiple shorter ones with line continuation char. Lose spurious quote. Declare local variable "tStartValue" to support strict compilation.

dunbarx committed:
Add a default handler to scrollbars

Two handlers should help the user get started with these controls. The progress bar is excluded. 

Changes on line 1106-1107

dunbarx committed:
Update revseutilities.livecodescript

livecode-vulcan commented:
📝 Hi @dunbarx, I haven't been able to verify that you've signed our [LiveCode Contributor's Agreement](https://livecode.com/account/developer/contribute).

If you have previously signed the Contributor's Agreement, I may not be able to detect it because you haven't [linked your GitHub account to your LiveCode account](https://livecode.com/account/developer/github).

Please see the [information for contributors](https://github.com/runrev/livecode/blob/develop/CONTRIBUTING.md) for more information.

dunbarx committed:
Update revseutilities.livecodescript

Bernd pointed out that scrollbars are not buttons. Who knew? So I created a new case statement for scrollbars. Also, the script local variable should start with "s", not "t". And I determined that an "end if" is needed in an if-then construction.

livecode-vulcan commented:
📝 Hi @dunbarx, I haven't been able to verify that you've signed our [LiveCode Contributor's Agreement](https://livecode.com/account/developer/contribute).

If you have previously signed the Contributor's Agreement, I may not be able to detect it because you haven't [linked your GitHub account to your LiveCode account](https://livecode.com/account/developer/github).

Please see the [information for contributors](https://github.com/runrev/livecode/blob/develop/CONTRIBUTING.md) for more information.

dunbarx committed:
Update revseutilities.livecodescript

Lose return before line "put the thumbPosition..." A little thing, but cleans up the final appearance of the code

dunbarx committed:
Update revseutilities.livecodescript

I originally copied the var "tIndent" from the case above. This is not referenced in the case for scrollbars. So I substitute two spaces explicitly.

dunbarx committed:
Update revseutilities.livecodescript

I apologize for the string of small changes. Why have two spaces on their own when two spaces will do in the literal?

dunbarx committed:
Update revseutilities.livecodescript

I hope this is the end. Moved variable "tIndent"  to the top of the handler since it is used in two case statements. Changed the style names to the correct ones, "Scale and "scrollbar"

livecodeali commented:
@dunbarx This just needs a bugfix note for the release notes.

dunbarx committed:
Create bugfix-17851.md

livecodeali commented:
@livecode-vulcan review ok 79303a3

livecode-vulcan commented:
💙 review by @livecodeali ok 79303a304d1516448e277c3f2259368b0ccec527
